docs/DeveloperGuide.md

```diff
@@ -494,6 +494,47 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 2b1. EstateEase shows error message regarding the IOException to the user.
       Use case ends.
 
+**Use case: Exit Application**
+
+**MSS:**
+
+1. User enters the 'exit' command.
+2. EstateEase immediately closes the application.
+
+**Extensions**
+
+* 1a. User enters an unrecognized command.
+    * 1a1. EstateEase displays a message "Unknown command".
+      Use case resumes from the previous step.
+
+**Use case: Linking Buyers to Sellers**
+
+**MSS:**
+
+1. User initiates the process of linking a buyer to sellers for a specific property.
+2. EstateEase validates the provided property information, buyer ID, and seller ID.
+3. If the provided information is valid, EstateEase proceeds to link the buyer to the specified seller(s) for the given property.
```

Comment:
`If the provided information is valid` 
I think you can remove this one as MSS assumes everything goes right.